ui/analyse/src/ctrl.ts

```diff
@@ -567,6 +567,13 @@ export default class AnalyseCtrl {
     if (this.study) this.study.deleteNode(path);
   }
 
+  deleteAllSidelines(path: Tree.Path): void {
+    if (this.study) return; // not implemented for studies yet
+    if (!confirm('Are you sure you want to delete all sidelines prior to the selected move?')) return;
```

Comment:
If you put this as an affirmation, like "Delete all variations prior to the selected move" then you can also use it as the `title` attribute of the button and it will only require one translation key. And it still works well in a confirm dialog.

Reply:
Ah, awesome pro tip. Will implement on the next pass 👍